TestDataGenerator/src/main/java/PatientProfile.java

```diff
@@ -0,0 +1,123 @@
+import com.github.javafaker.Faker;
+
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+public class PatientProfile {
+
+    Faker faker = new Faker(new Locale("en-IND"));;
+    protected String[] createPatientProfile()
+    {
+        String villageDistrict = getTownCityName();
+
+        String[] item = {"x", getRegistrationDate(), getFirstName(), getMiddleName(),
+                getLastName(), getGender(), getBirthDate(), villageDistrict, getTehsilName(), villageDistrict,
+                getStateName()};
+        return item;
+    }
+
+    protected List<String[]> getPatientProfileList(int count)
+    {
+        List<String[]> entries = new ArrayList<>();
+        String[] header = {"Registration Number","Registration Date","First Name","Middle Name","Last Name","Gender",
+        "Birth Date", "Address.Village", "Address.Tehsil", "Address.District", "Address.State"};
```

Comment:
Just note that this is dependant on Bahmni server configuration. for instance: "default-config" may specify these fields... in AddressHierarchy setup. But the "clinic-config" based Bahmni server setup might have different address hierarchy.. and in that case this script won't work. 

Please see if u can mention this information in your folder README.. to indicate the assumputions you made or the config folder this script runs with.